test/test_ringbuffer.toit

```diff
@@ -4,7 +4,11 @@ import ringbuffer show *
 class TestRingbuffer implements TestCase:
 
   run:
+    // test_ringbuffer_empty  //@davidg238: not calling assertException correctly?
```

Comment:
Yes - the assertException method is a bit confusing because Exception can be anything. You basically call the method first and catch the exception, then you check the exception like:

```
exception := catch:
      ringbuffer.average
assertException exception
```
I just realized that I used it incorrectly myself in the test for the limit. I think the assertException can be done smarter where you give it a block which the method will run and check

Reply:
It looked like the test framework (love the name) was young, so I flagged it rather than attempt to fix it.  A block would be preferable to simplify use.